types/office-js-preview/index.d.ts

```diff
@@ -38219,6 +38219,22 @@ declare namespace Excel {
          * [Api set: ExcelApi 1.1]
          */
         delete(): void;
+        /**
+         * Gets the data source of the whole chart.
+         *
+         * @remarks
+         * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+         * @beta
+         */
+         getDataRange(): OfficeExtension.ClientResult<string>;
+         /**
+          * Gets the data source of the whole chart. If the current data source is not supported yet, this method will not throw exception.
```

Comment:
I believe the descriptions for getDataRange and getDataRangeOrNullObject are reversed. The pattern for our get* and get*OrNullObject methods is that the former throws an exception if the entity isn't there and the latter does NOT throw an exception. Instead, it returns a proxy object whose isNullObject property is set to true.

Reply:
Thanks, Ricky! This is very helpful. I will confirm with PM/eng.

Reply:
I updated this based on your feedback + internal discussion. Please take a look when you have a chance. Thanks!